README.md

```diff
@@ -71,7 +75,7 @@
 - [Jonathan Beri on Embedded.fm Episode 222](https://www.embedded.fm/episodes/222)
 - [Brandon Satrom on the MacroFab Engineering Podcast Episode 122](https://macrofab.com/blog/mep-ep122-brandon-satrom-lanot/)
 
-## Useful Docs & FAQs
```

Comment:
I don't want to create categories of one, so let's move this under "Useful Docs & FAQs" and change the heading to "Useful Tools, Docs & FAQs"

Reply:
Are you saying I should put po-util under "Useful Tools, Docs & FAQs" and remove the "Community Tools and SDKs" ?

Reply:
Yes